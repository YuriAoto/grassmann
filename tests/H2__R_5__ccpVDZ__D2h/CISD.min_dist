dGr - optimise the distance in the Grassmannian
Yuri Aoto - 2018, 2019

Current git commit: 2aa059ca687f5285c7df78a9478091e3c6ffd720

Directory:
/home/yuriaoto/Documents/Codes/min_dist_Gr/tests/H2__R_5__ccpVDZ__D2h

Command:
../../dGr --loglevel 0 CISD.out

From command line and environment:
External wave function, |extWF>: CISD.out
Orbital basis of |extWF>: CISD.out
Hartree-Fock orbitals, |min E>: CISD.out
Using the optimiser based on the geometry of the Grassmannian. See:
   P-A Absil et. al, Acta Applicandae Mathematicae 80, 199-220, 2004.
State: 
Log level: NOTSET
qpy job ID: not in qpy

Starting at 21 Oct 2019 - 09:23

Using |WFref> (the reference of |extWF>) as |min E>:
|<WFref|extWF>| =   0.78181588 ; D(WFref, extWF) =   0.66058175
Using the reference determinant (identity) as initial guess for U.
Dimension of orbital space: alpha/beta: [ 3 1 1 0 3 1 1 0 ]
Number of alpha and beta electrons: 1 1

Starting optimisation
------------------------------
it.    f            |eta|        |C|          time in iteration
0      0.78181588   ---          ---          ---
1      0.79089561   0.15396558   0.11935595   0:00:00.097001
2      0.79089783   0.00236626   0.00187733   0:00:00.089911
3      0.79089783   0.00000001   0.00000001   0:00:00.089636
------------------------------
Iterations converged!
WARNING: Unknown number of positive eigenvalue(s) of Hessian.
|<min D|extWF>| =   0.79089783 ; D(min D, extWF) =   0.64668720
|<min E|min D>| =   0.98855277 ; D(min E, min D) =   0.15130914

Ending at 21 Oct 2019 - 09:23
Total time: 0:00:00.301922
