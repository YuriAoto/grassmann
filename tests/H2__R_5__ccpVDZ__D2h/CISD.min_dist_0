dGr - optimise the distance in the Grassmannian
Yuri Aoto - 2018, 2019, 2020

Current git commit: 1d5924d05472d0c764545b06dde602c5e2b664fb

Directory:
/home/yuriaoto/Documents/Codes/min_dist_Gr/tests/H2__R_5__ccpVDZ__D2h

Command:
../../dGr \
CISD.out 

From command line and environment:
External wave function, |extWF>: CISD.out
Orbital basis of |extWF>: CISD.out
Hartree-Fock orbitals, |min E>: CISD.out
Using the optimiser based on the geometry of the Grassmannian. See:
   P-A Absil et. al, Acta Applicandae Mathematicae 80, 199-220, 2004.
State: 
Log level: WARNING
qpy job ID: not in qpy

Starting at 25 Mar 2020 - 15:42

Using algorithm specific for CISD wave function.
Using |WFref> (the reference of |extWF>) as |min E>.
|<WFref|extWF>| =   0.78181588 ; D(WFref, extWF) =   0.66058175
Using the reference determinant (identity) as initial guess for U.
Dimension of orbital space: alpha/beta: [ 3 1 1 0 3 1 1 0 ]
Number of alpha and beta electrons: 1 1
Occupation of reference wave function: alpha/beta: [ 1 0 0 0 0 0 0 0 ]

Starting optimisation
------------------------------
it.    f            |eta|        |C|          time in iteration
0      0.78181588   0.10887010   0.08439740   0:00:00.000872
1      0.79089561   0.00167320   0.00132747   0:00:00.000637
2      0.79089783   0.00000001   0.00000000   0:00:00.000590
------------------------------
Iterations converged!
WARNING: Unknown number of positive eigenvalue(s) of Hessian.
|<min D|extWF>| =   0.79089783 ; D(min D, extWF) =   0.64668720
|<min E|min D>| =   0.98855277 ; D(min E, min D) =   0.15130914

Ending at 25 Mar 2020 - 15:42
Total time: 0:00:00.041372
