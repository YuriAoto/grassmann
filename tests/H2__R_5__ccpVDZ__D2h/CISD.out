
 Primary working directories    : /scr/yuriaoto/job.022374/molpro.pOOYob2kcI
 Secondary working directories  : /scr/yuriaoto/job.022374/molpro.pOOYob2kcI
 Wavefunction directory         : /scr/yuriaoto/job.022374/
 Main file repository           : /scr/yuriaoto/job.022374/molpro.pOOYob2kcI/

 id        : yuriaoto

 Nodes             nprocs
 t-rex.ufabc.int.br    1

 Using customized tuning parameters: mindgm=8; mindgv=16; mindgc=1; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (933), CPU time= 0.00 sec
 ***,
 memory,40,m
 gprint,orbital,basis,civector
 gthresh,thrprint=0,printci=0.0
 file,3,wfu.dat,old
 
 {cisd,CC_NORM_MAX=10.0
 thresh,printci=0.0,ENERGY=1.0E-10,COEFF=1.0E-12
 print,ref=1
 core,0
 wf,2,1,0;
 }
 
 Commands  initialized (760), CPU time= 0.01 sec, 617 directives.
 Default parameters read. Elapsed time= 0.05 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2019.2 linked Abr  7 2019 15:10:35


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit serial version                                                                  DATE: 16-Oct-19          TIME: 09:34:35  
 **********************************************************************************************************************************

 SHA1:             3a8a58c585c929645742fb5a1b7541a4ad1da699
 **********************************************************************************************************************************


 THRESHOLDS:

 ZERO    =  1.00D-12  ONEINT  =  1.00D-12  TWOINT  =  1.00D-11  PREFAC  =  1.00D-14  XXXXXX  =  1.00D-09  EORDER  =  1.00D-04
 ENERGY  =  0.00D+00  ETEST   =  0.00D+00  EDENS   =  0.00D+00  THRDEDEF=  1.00D-06  GRADIENT=  1.00D-02  STEP    =  1.00D-03
 ORBITAL =  1.00D-05  CIVEC   =  1.00D-05  COEFF   =  1.00D-04  PRINTCI =  0.00D+00  PUNCHCI =  9.90D+01  OPTGRAD =  3.00D-04
 OPTENERG=  1.00D-06  OPTSTEP =  3.00D-04  THRGRAD =  1.00D-10  COMPRESS=  1.00D-11  VARMIN  =  1.00D-07  VARMAX  =  1.00D-03
 THRDOUB =  0.00D+00  THRDIV  =  1.00D-05  THRRED  =  1.00D-07  THRPSP  =  1.00D+00  THRDC   =  1.00D-10  THRCS   =  1.00D-10
 THRNRM  =  1.00D-08  THREQ   =  0.00D+00  THRDE   =  1.00D+00  THRREF  =  1.00D-05  SPARFAC =  1.00D+00  THRDLP  =  1.00D-07
 THRDIA  =  1.00D-10  THRDLS  =  1.00D-07  THRGPS  =  0.00D+00  THRKEX  =  0.00D+00  THRDIS  =  2.00D-01  THRVAR  =  1.00D-10
 THRLOC  =  1.00D-06  THRGAP  =  1.00D-06  THRLOCT = -1.00D+00  THRGAPT = -1.00D+00  THRORB  =  1.00D-06  THRMLTP =  0.00D+00
 THRCPQCI=  1.00D-10  KEXTA   =  0.00D+00  THRCOARS=  0.00D+00  SYMTOL  =  1.00D-06  GRADTOL =  1.00D-06  THROVL  =  1.00D-08
 THRORTH =  1.00D-08  GRID    =  1.00D-06  GRIDMAX =  1.00D-03  DTMAX   =  0.00D+00  THROCCDE=  1.00D-01  THROCCDE=  1.00D-01
 THRPRINT=  0.00D+00


 Variable memory set to   40000000 words,  buffer space 



 Permanent file  3  wfu.dat  assigned.  Implementation=df    Size= 14.08 MB


 PROGRAM * RESTART 


 Reading variables from file 3

 _FNUC           =      0.00000000
 _BASIS          =    CC-PVDZ
 _NELEC          =      2.00000000
 _PROGRAM        =    RHF-SCF
 _DMX_SCF        =      0.00000000
 _DMY_SCF        =      0.00000000
 _DMZ_SCF        =      0.00000000
 _HOMO           =      1.10000000
 _EHOMO          =     -0.34086982
 _LUMO           =      1.50000000
 _ELUMO          =     -0.09520302
 _ENERGY         =     -0.85242437
 _ENERGY_METHOD  =    RHF
 _ENERGY_BASIS   =    cc-pVDZ
 _ENUC           =      0.20000000
 _EPS1(1:3)      =     -0.34086982     0.61305106     1.46113314
 _EPS2           =      1.46277864
 _EPS3           =      1.46277864
 _EPS5(1:3)      =     -0.09520302     0.67276230     1.50154450
 _EPS6           =      1.46773963
 _EPS7           =      1.46773963
 _IPROC_MPP      =      0.00000000
 _IPROC_MPPX     =      0.00000000
 _ORBITAL        =   2100.30000000
 _STATUS         =      1.00000000
 _LASTORB        =    RHF
 _OUTPUT         =    /scr/yuriaoto/job.022372/RHF.xml
 _PGROUP         =    D2h 
 _TIME           =    09:34:25
 R               =      5.00000000

 Geometry written to block  1 of record 700


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL   RESTART
 CPU TIMES  *         0.12      0.01
 REAL TIME  *         0.15 SEC
 DISK USED  *        26.84 MB
 **********************************************************************************************************************************


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry H      S cc-pVDZ              selected for orbital group  1
 Library entry H      P cc-pVDZ              selected for orbital group  1


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Linear
 User-specified symmetry elements: X,Y,Z
 Symmetry elements: X,Y,Z
 Rotational constants:     143.2419517    143.2419517      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  H1      1.00    0.000000000    0.000000000   -2.500000000
   2  H2      1.00    0.000000000    0.000000000    2.500000000


 BASIS DATA

   Nr Sym  Nuc  Type         Exponents   Contraction coefficients

    1.1 Ag    1  1s          13.010000     0.019685    0.000000
    2.1 Ag    2  1s           1.962000     0.137977    0.000000
                              0.444600     0.478148    0.000000
                              0.122000     0.501240    1.000000
    3.1 Ag    1  2pz          0.727000     1.000000
              2 -2pz   
    1.2 B3u   1  2px          0.727000     1.000000
              2  2px   
    1.3 B2u   1  2py          0.727000     1.000000
              2  2py   
    1.5 B1u   1  1s          13.010000     0.019685    0.000000
    2.5 B1u   2 -1s           1.962000     0.137977    0.000000
                              0.444600     0.478148    0.000000
                              0.122000     0.501240    1.000000
    3.5 B1u   1  2pz          0.727000     1.000000
              2  2pz   
    1.6 B2g   1  2px          0.727000     1.000000
              2 -2px   
    1.7 B3g   1  2py          0.727000     1.000000
              2 -2py   

 NUCLEAR CHARGE:                    2
 NUMBER OF PRIMITIVE AOS:          14
 NUMBER OF SYMMETRY AOS:           14
 NUMBER OF CONTRACTIONS:           10   (    3Ag  +    1B3u +    1B2u +    0B1g +    3B1u +    1B2g +    1B3g +    0Au  )
 NUMBER OF CORE ORBITALS:           0   (    0Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF VALENCE ORBITALS:        2   (    1Ag  +    0B3u +    0B2u +    0B1g +    1B1u +    0B2g +    0B3g +    0Au  )


 NUCLEAR REPULSION ENERGY    0.20000000

 Eigenvalues of metric

         1 0.104E+00 0.999E+00 0.221E+01
         2 0.100E+01
         3 0.100E+01
         5 0.832E-01 0.993E+00 0.161E+01
         6 0.100E+01
         7 0.100E+01


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:        373.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:        373      RECORD LENGTH: 524288

 Memory used in sort:       0.56 MW

 SORT1 READ         776. AND WROTE         373. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC
 SORT2 READ         373. AND WROTE         373. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC

 FILE SIZES:   FILE 1:    30.2 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:     34.4 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       29.94       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       INT   RESTART
 CPU TIMES  *         0.27      0.15      0.01
 REAL TIME  *         0.43 SEC
 DISK USED  *        49.42 MB
 **********************************************************************************************************************************


 PROGRAM * CISD (Closed-shell CI(SD))     Authors: C. Hampel, H.-J. Werner, 1991, M. Deegan, P.J. Knowles, 1992


 Input parameters for THRESH:

 ENERGY   =    1.00D-10
 COEFF    =    1.00D-12
 PRINTCI  =    0.000000
 THRPRINT =    0.000000

 Convergence thresholds:  THRVAR = 1.00D-24  THRDEN = 1.00D-10

 Number of closed-shell orbitals:   1 (   1   0   0   0   0   0   0 )
 Number of external orbitals:       9 (   2   1   1   0   3   1   1 )

 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)

 Number of N-1 electron functions:               1
 Number of N-2 electron functions:               1
 Number of singly external CSFs:                 2
 Number of doubly external CSFs:                13
 Total number of CSFs:                          16

 Length of J-op  integral file:               0.00 MB
 Length of K-op  integral file:               0.03 MB

 Integral transformation finished. Total CPU:   0.00 sec, npass=  1  Memory used:   0.07 MW

 Reference energy:                     -0.85242437

 MP2 singlet pair energy:              -0.07826033
 MP2 triplet pair energy:               0.00000000
 MP2 correlation energy:               -0.07826033
 MP2 total energy:                     -0.93068469

 SCS-MP2 correlation energy:           -0.09391239   (PS=  1.200000  PT=  0.333333)
 SCS-MP2 total energy:                 -0.94633676

 ITER.      SQ.NORM     CORR.ENERGY   TOTAL ENERGY   ENERGY CHANGE        DEN1      VAR(S)    VAR(P)  DIIS     TIME  TIME/IT
   1      1.26455168    -0.12887029    -0.98129466    -0.05060996    -0.02343615  0.41D-02  0.22D-01  1  1     0.00     0.00
   2      1.49898047    -0.13529261    -0.98771698    -0.00642232    -0.03174423  0.41D-05  0.11D-02  2  2     0.00     0.00
   3      1.72608522    -0.14799130    -1.00041567    -0.01269869    -0.02531330  0.52D-04  0.14D-03  3  3     0.01     0.00
   4      1.67176433    -0.14870673    -1.00113110    -0.00071543     0.00586505  0.23D-05  0.10D-05  4  4     0.01     0.00
   5      1.64061241    -0.14901665    -1.00144101    -0.00030991     0.00330527  0.33D-07  0.99D-07  5  5     0.01     0.00
   6      1.63846093    -0.14907829    -1.00150265    -0.00006164     0.00025260  0.19D-07  0.41D-08  6  6     0.01     0.00
   7      1.63662656    -0.14907917    -1.00150353    -0.00000088     0.00018818  0.78D-12  0.49D-09  6  2     0.01     0.00
   8      1.63597480    -0.14907946    -1.00150382    -0.00000029     0.00006928  0.51D-11  0.93D-11  6  1     0.01     0.00
   9      1.63603544    -0.14907948    -1.00150384    -0.00000002    -0.00000652  0.17D-13  0.64D-13  6  3     0.01     0.00
  10      1.63602877    -0.14907948    -1.00150384    -0.00000000     0.00000072  0.12D-14  0.51D-14  6  4     0.01     0.00
  11      1.63602923    -0.14907948    -1.00150384    -0.00000000    -0.00000005  0.36D-16  0.68D-16  6  6     0.01     0.00
  12      1.63602912    -0.14907948    -1.00150384    -0.00000000     0.00000001  0.11D-16  0.30D-17  6  5     0.01     0.00
  13      1.63602912    -0.14907948    -1.00150384    -0.00000000     0.00000000  0.24D-18  0.18D-18  6  2     0.01     0.00
  14      1.63602912    -0.14907948    -1.00150384    -0.00000000     0.00000000  0.14D-20  0.11D-19  6  1     0.01     0.00
  15      1.63602912    -0.14907948    -1.00150384     0.00000000     0.00000000  0.19D-23  0.31D-22  6  6     0.01     0.00
  16      1.63602912    -0.14907948    -1.00150384     0.00000000    -0.00000000  0.62D-26  0.20D-23  6  3     0.01     0.00
  17      1.63602912    -0.14907948    -1.00150384     0.00000000     0.00000000  0.19D-26  0.20D-26  6  4     0.01     0.00


 ! DIPOLE MOMENTS:                      0.00000000     0.00000000     0.00000000



 Norm of t1 vector:      0.15266504      S-energy:    -0.00212376      T1 diagnostic:  0.07633252

 Singles amplitudes (print threshold =  0.000E+00):

         I         SYM. A    A   T(IA)

         1         1         1     -0.10780149
         1         1         2     -0.00566969

 Doubles amplitudes (print threshold =  0.000E+00):

         I         J         SYM. A    SYM. B    A         B      T(IJ, AB)

         1         1         1         1         1         1      0.00839802
         1         1         1         1         1         2      0.00083091
         1         1         1         1         2         1      0.00083091
         1         1         1         1         2         2     -0.00546493
         1         1         2         2         1         1     -0.00374441
         1         1         3         3         1         1     -0.00374441
         1         1         5         5         1         1     -0.76921165
         1         1         5         5         1         2      0.10150132
         1         1         5         5         1         3     -0.00563809
         1         1         5         5         2         1      0.10150132
         1         1         5         5         2         2     -0.01518971
         1         1         5         5         2         3      0.00068753
         1         1         5         5         3         1     -0.00563809
         1         1         5         5         3         2      0.00068753
         1         1         5         5         3         3     -0.00120278
         1         1         6         6         1         1     -0.00148179
         1         1         7         7         1         1     -0.00148179


 RESULTS
 =======

  Reference energy                     -0.852424365607
  CISD singlet pair energy             -0.146955713329
  CISD triplet pair energy              0.000000000000
  CISD correlation energy              -0.149079476043

 !CISD total energy                    -1.001503841650

 Timing summary (sec):

 STEP                 CPU(USER)    SYS     CPU(TOT)    WALL
 CCSD iterations         0.01      0.01      0.02      0.03

 Program statistics:

 Available memory in ccsd:                39999196
 Min. memory needed in ccsd:                   382
 Max. memory used in ccsd:                     382
 Max. memory used in cckext:                 33022 (17 integral passes)



 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      20       29.94       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700     1380   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER     JKOP   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL      CISD       INT   RESTART
 CPU TIMES  *         0.31      0.04      0.15      0.01
 REAL TIME  *         0.48 SEC
 DISK USED  *        49.42 MB
 **********************************************************************************************************************************
 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)
 Occupation numbers read from record     2100.3  Type=RHF/RHF (state 1.1)
 Orbital energies read from record       2100.3  Type=RHF/CANONICAL (state 1.1)
 Redundancy group numbers read from rec  2100.3  Type=RHF/RHF (state 1.1)

 CISD/cc-pVDZ energy=     -1.001503841650

            CISD
     -1.00150384
 **********************************************************************************************************************************
 Molpro calculation terminated
